### PR TITLE
CI: Update RS/FEM scope in test-spec.yaml

### PR DESCRIPTION
**Pull request by dawidprzybylo**

### CODE REVIEW

File: .github/test-spec.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   - "nrf_modem/**/*"
 
 "CI-lwm2m-test":
-  - "nrf_modem/**/*"
+  - "nrf_modem/**/*"f
```
Comment (e-rk) [inline .github/test-spec.yml:14]:
```suggestion
  - "nrf_modem/**/*"
```

File: .github/test-spec.yml [hunk 2]
```diff
@@ -51,15 +51,19 @@
   - "nrf_security/**/*"
 
 "CI-rs-test":
+  - "mpsl/CmakeLists.txt"
```
Comment (e-rk) [inline .github/test-spec.yml:54]:
```suggestion
  - "mpsl/CMakeLists.txt"
```

**Review by e-rk — CHANGES_REQUESTED** [review]

File: .github/test-spec.yml (hunk 1 above)
Comment (dawidprzybylo) [reply .github/test-spec.yml]: Fixed :)

File: .github/test-spec.yml (hunk 2 above)
Comment (dawidprzybylo) [reply .github/test-spec.yml]: Fixed :)

File: .github/test-spec.yml [hunk 3]
```diff
@@ -51,15 +51,19 @@
...
   - "mpsl/include/**/*"
   - "mpsl/lib/**/*"
+  - "softdevice_controller/CmakeLists.txt"
```
[the remark this answers is not in the document]
Comment (dawidprzybylo) [reply .github/test-spec.yml]: Fixed :)

File: .github/test-spec.yml [hunk 4]
```diff
@@ -51,15 +51,19 @@
...
   - "softdevice_controller/include/**/*"
   - "softdevice_controller/lib/**/*"
   - "nrf_802154/**/*"
 
 "CI-fem-test":
+  - "mpsl/CmakeLists.txt"
```
[the remark this answers is not in the document]
Comment (dawidprzybylo) [reply .github/test-spec.yml]: Fixed :)

File: .github/test-spec.yml [hunk 5]
```diff
@@ -51,15 +51,19 @@
...
   - "mpsl/include/**/*"
   - "mpsl/lib/**/*"
+  - "softdevice_controller/CmakeLists.txt"
```
[the remark this answers is not in the document]
Comment (dawidprzybylo) [reply .github/test-spec.yml]: Fixed :)

**Review by e-rk — APPROVED** [review]

**Review by jciupis — APPROVED** [review]

**Review by czeslawmakarski — APPROVED** [review]

**Pull request merged**